src/components/reactHooks/context.jsx

```diff
@@ -0,0 +1,38 @@
+import React, { useState, createContext } from "react";
+import Footer from "../leyout/footer";
+import Header from "../leyout/header";
```

Comment:
typo: should be "layout"
```suggestion
import Footer from "../layout/footer";
import Header from "../layout/header";
```

Reply:
Done